adagrams/game.py

```diff
@@ -1,11 +1,104 @@
+import random
+from re import A
```

Comment:
Oops! Looks like VS Code accidentally auto-imported something you didn't want here. Make sure to double-check your files in the future and remove accidental imports like these.